Workbooks/Workloads/Onboard SQL Azure Insights/Onboard SQL Azure Insights.workbook

```diff
@@ -438,6 +296,77 @@
                   },
                   "name": "remotemachineparams"
                 },
+                {
+                  "type": 1,
+                  "content": {
+                    "json": "#### Setup SQL database monitoring\r\n    \r\nSQL Server monitoring data collection is specified by Telegraf configuration below. Update the `servers` section with the correct connection strings to access your SQL databases. You can also customize your data collection settings using the config. Learn about [SQL Telegraf settings](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/sqlserver#configuration)."
+                  },
+                  "name": "text - 4",
+                  "styleSettings": {
+                    "margin": "20px 0 0 0"
+                  }
+                },
+                {
+                  "type": 9,
+                  "content": {
+                    "version": "KqlParameterItem/1.0",
+                    "crossComponentResources": [
+                      "value::all"
+                    ],
+                    "parameters": [
+                      {
+                        "id": "573cff77-860f-4b27-bfa4-1ca675a13b98",
+                        "version": "KqlParameterItem/1.0",
+                        "name": "SqlTelegrafConfig",
+                        "label": "Sql Telegraf config",
+                        "type": 1,
+                        "query": "Resources\r\n| take 1\r\n| project toml = strcat('[[inputs.sqlserver]]\\r\\n  servers = [\\r\\n    {SQLServer}\\r\\n  ]\\r\\n\\r\\n  query_version = 2\\r\\n  azuredb=true\\r\\n  exclude_query = [ \\'Schedulers\\' , \\'SqlRequests\\']\\r\\n\\r\\n  [inputs.sqlserver.tags]\\r\\n    insight = \"sqlaz\"')",

```

Comment:
I wonder if the insight tags should not be 'sqlaz' be ‘sql-azure’ like in the insight workbook.

Reply:
yes, it should be 'sql-azure', will update.

Reply:
fixed